src/main/scala/ch/openolitor/buchhaltung/BuchhaltungCommandHandler.scala

```diff
@@ -209,6 +217,26 @@ trait BuchhaltungCommandHandler extends CommandHandler with BuchhaltungDBMapping
         Success(successfuls flatMap (_.get))
       }
 
+    case SendEmailToInvoicesSubscribersCommand(personId, subject, body, ids, attachInvoice) => idFactory => meta =>
+      DB readOnly { implicit session =>
+        if (checkTemplateInvoice(body, subject, ids)) {
+          val events = ids flatMap { rechnungId: RechnungId =>
+            buchhaltungReadRepository.getById(rechnungMapping, rechnungId) map { rechnung =>
+              buchhaltungReadRepository.getPerson(rechnung.id) map { person =>
+                val mailContext = RechnungMailContext(person, rechnung)
+                rechnung.fileStoreId match {
+                  case Some(invoiceReference) => DefaultResultingEvent(factory => SendEmailToInvoiceSubscribersEvent(factory.newMetadata(), subject, body, person, Some(invoiceReference), mailContext))
```

Comment:
Why the match? Can't you use the fileStoreId directly? 
DefaultResultingEvent(factory => SendEmailToInvoiceSubscribersEvent(factory.newMetadata(), subject, body, person, rechnung.fileStoreId, mailContext))

Reply:
You are right!